### PR TITLE
[14.0] [IMP] sale_commission_geo_assign_product_domain: add tree view

### PR DESCRIPTION
**Pull request by renda-dev**
Added a tree view for `res.partner.commission.group`

### CODE REVIEW

File: sale_commission_geo_assign_product_domain/views/res_partner_commission_group_view.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                />
+            </search>
+        </field>
+    </record>
+
+    <record id="res_partner_commission_group_action" model="ir.actions.act_window">
+        <field name="name">Partner Commission Group</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">res.partner.commission.group</field>
+        <field name="binding_view_types">tree</field>
```
Comment (HekkiMelody) [inline sale_commission_geo_assign_product_domain/views/res_partner_commission_group_view.xml]: **issue**: `binding_view_types` is used for actions that can be called from the `Action` menù (like Duplicate and Archive, for example)

Here I think you want to set `view_mode` instead.

File: sale_commission_geo_assign_product_domain/views/res_partner_commission_group_view.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                    domain="[]"
+                    context="{'group_by': 'partner_id'}"
+                />
+            </search>
+        </field>
+    </record>
+
+    <record id="res_partner_commission_group_action" model="ir.actions.act_window">
+        <field name="name">Partner Commission Group</field>
+        <field name="type">ir.actions.act_window</field>
```
Comment (HekkiMelody) [inline sale_commission_geo_assign_product_domain/views/res_partner_commission_group_view.xml]: **chore**: Setting this `type` is not necessary, it's already implicit from the `model="ir.actions.act_window"`

**Review by HekkiMelody — CHANGES_REQUESTED** [review]
A couple of notes before it can be approved.

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

**Review by HekkiMelody — APPROVED** [review]
Code review, LGTM

**Pull request merged**